ui/pages/confirmations/components/confirm/pluggable-section/pluggable-section.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { ReactComponentLike } from 'prop-types';
+import { useSelector } from 'react-redux';
+
+import { currentConfirmationSelector } from '../../../selectors';
+
+// Components to be plugged into confirmation page can be added to the array below
+const PluggedInSections: ReactComponentLike[] = [];
+
+const PluggableSection = () => {
```

Comment:
Given the intent of all these nested components, whether snaps or anything else in future, is to add insights to the confirmation. Would it be more explicit to call this `InsightContainer` or even just `Insights` to match the other components such as `Title`, `Header`, and `Info`?

Reply:
I think pluggable section is more generic name, components we add can be anything - may be even insight I guess.